tools/oeutil/host/generate_evidence.cpp

```diff
@@ -647,6 +897,88 @@ oe_result_t get_oe_report_from_certificate(
     return result;
 }
 
+oe_result_t generate_certificate(
+    oe_enclave_t* enclave,
+    uint8_t* private_key,
+    size_t private_key_size,
+    uint8_t* public_key,
+    size_t public_key_size,
+    const char* certificate_filename,
+    bool verify,
+    bool verbose)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    oe_result_t ecall_result;
+    unsigned char* cert = nullptr;
+    size_t cert_size = 0;
```

Comment:
```suggestion
    unsigned char* certificate = nullptr;
    size_t certificate_size = 0;
```
avoid abbreviations per OE style guidance.  Match "certificate" as used in line 906